src/tests/ftest/pool/create_all_hw.py

```diff
@@ -244,37 +232,33 @@ def test_two_pools(self):
-                abs(self.scm_avail_bytes - tier_bytes[1][0]), self.delta_bytes,
-                "Invalid SCM size: want={}, got={}, delta={}".format(
-                    self.scm_avail_bytes, tier_bytes[1][0], self.delta_bytes))
-        self.assertLessEqual(
-                abs(self.smd_avail_bytes - tier_bytes[1][1]), self.delta_bytes,
-                "Invalid SMD size: want={}, got={}, delta={}".format(
-                    self.smd_avail_bytes, tier_bytes[1][1], self.delta_bytes))
+        self.assertLessEqual(abs(self.scm_avail_bytes - tier_bytes[1][0]), self.delta_bytes,
+                             "Invalid SCM size: want={}, got={}, delta={}".format(
+                             self.scm_avail_bytes, tier_bytes[1][0], self.delta_bytes))
+        self.assertLessEqual(abs(self.smd_avail_bytes - tier_bytes[1][1]), self.delta_bytes,
+                             "Invalid SMD size: want={}, got={}, delta={}".format(
+                             self.smd_avail_bytes, tier_bytes[1][1], self.delta_bytes))
 
         self.log.info("Checking size of the second pool with the size of the first pool")
         scm_delta_bytes = self.ranks_size * self.max_scm_metadata_bytes + self.delta_bytes
-        self.assertLessEqual(
-                abs(tier_bytes[0][0] - tier_bytes[1][0]), scm_delta_bytes,
-                "Invalid SCM size: want={}, got={}, delta={}".format(
-                    tier_bytes[0][0], tier_bytes[1][0], self.delta_bytes))
+        self.assertLessEqual(abs(tier_bytes[0][0] - tier_bytes[1][0]), scm_delta_bytes,
+                             "Invalid SCM size: want={}, got={}, delta={}".format(
+                             tier_bytes[0][0], tier_bytes[1][0], self.delta_bytes))
         smd_delta_bytes = self.ranks_size * max(1, self.engine_target_size - 1)
         smd_delta_bytes *= self.smd_cluster_bytes
         smd_delta_bytes += self.delta_bytes
-        self.assertLessEqual(
-                abs(tier_bytes[0][1] - tier_bytes[1][1]), smd_delta_bytes,
-                "Invalid SMD size: want={}, got={}, delta={}".format(
-                    tier_bytes[0][1], tier_bytes[1][1], self.delta_bytes))
+        self.assertLessEqual(abs(tier_bytes[0][1] - tier_bytes[1][1]), smd_delta_bytes,
+                             "Invalid SMD size: want={}, got={}, delta={}".format(
+                             tier_bytes[0][1], tier_bytes[1][1], self.delta_bytes))
 
         self.scm_avail_bytes, self.smd_avail_bytes = self.get_pool_available_bytes()
 
         self.log.info("Checking size of available storage after the creation of the second pool")
         self.assertLessEqual(
-                self.scm_avail_bytes, self.delta_bytes,
-                "Invalid SCM size: want=0, got={}, delta={}".format(
-                    self.scm_avail_bytes, self.delta_bytes))
+            self.scm_avail_bytes, self.delta_bytes,
+            "Invalid SCM size: want=0, got={}, delta={}".format(
+                self.scm_avail_bytes, self.delta_bytes))
```

Comment:
E.g. this format is similar to many that were changed above

Reply:
I don't get it either, like you say they look very similar.